Refit.Tests/InterfaceStubGenerator.cs

```diff
@@ -3,22 +3,29 @@
 using System.IO;
 using System.Linq;
 using System.Reflection;
+using System.Text;
 using System.Threading.Tasks;
 
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CSharp;
-using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Testing;
+using Microsoft.CodeAnalysis.Text;
 
-using Refit; // InterfaceStubGenerator looks for this
 using Refit.Generator;
+
 using Xunit;
 
 using Task = System.Threading.Tasks.Task;
+using VerifyCS = Refit.Tests.CSharpSourceGeneratorVerifier<Refit.Generator.InterfaceStubGenerator>;
 
 namespace Refit.Tests
 {
     public class InterfaceStubGeneratorTests
     {
+        static readonly MetadataReference RefitAssembly = MetadataReference.CreateFromFile(
+            typeof(GetAttribute).Assembly.Location,
+            documentation: XmlDocumentationProvider.CreateFromFile(Path.ChangeExtension(typeof(GetAttribute).Assembly.Location, ".xml")));
```

Comment:
📝 This would be better if we could reference a NuGet package (where it would automatically detect the proper TFM to reference and include the corresponding dependencies), but that would restrict the tests to using a Refit build available on NuGet.